fingertip/plugins/unseal.py

```diff
@@ -8,4 +8,5 @@ def main(m):
         m.sealed = False
         with m:
             m.hooks.unseal()
+            m.hooks.disrupt()
```

Comment:
I wonder why is that one needed, it should be called anyway if `.apply('unseal')` was outside `with`.

Reply:
Let me re-check if I did not have some outdated unsealed machines cached.

Reply:
Can you clarify where do you think it shuold call the disrupt() when applying?